src/oc/lib/jwt.clj

```diff
@@ -125,16 +125,27 @@
                       t/hours t/from-now .getMillis)]
     (assoc payload :expire expire-by)))
 
+(defn encode [payload passphrase]
+  (-> payload
+      jwt/jwt
+      (jwt/sign :HS256 passphrase)
+      jwt/to-str))
+
+(defn generate-id-token [claims passphrase]
+  (encode {:secure_uuid (:secure-uuid claims)
+           :org_id (:org-uuid claims)
+           :user_id (:user-id claims)
+           :avatar_url (:avatar-url claims)
+           :team_id (:team-id claims)}
```

Comment:
@thepug there is a particular reason to use the underscore instead of the dash here? I think keeping the same key names it's going to be helpful.
Right now we are not reading this in web but one day we could be and having the same key names as the JWT is going to be better i think.

Reply:
The reason is that it eventually gets used in javascript, but not a real reason. They can be changed.

Reply:
Yeah let's change them to use dash since we have the keys that way in all places. WYT?